powerauth-admin/pom.xml

```diff
@@ -20,12 +20,6 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
-            <exclusions>
```

Comment:
I think this change was intended as temporary - the Log4j issue was not really affecting us at the time, but since we were unable to bump the Spring Boot dependency quickly, we would get false positives in automated scans.